Assets/00_Content/Scripts/Rounds/RoundController.cs

```diff
@@ -71,33 +79,36 @@ public class RoundController : SingletonBehaviour<RoundController> {
 			VikingController.Instance.StatScaling = new VikingScaling(difficulty, currentRound);
 		}
 
+		private void DisableGamePlay() {
+			VikingController.Instance.CanSpawn = false;
+
+			foreach (PlayerComponent player in PlayerManager.Instance.Players) {
+				PlayerInput playerInput = player.GetComponent<PlayerInput>();
+				playerInput.SwitchCurrentActionMap("UI");
+			}
+		}
+
 		private void ShowScoreCard() {
 			if (Tavern.Instance != null && Tavern.Instance.Money < requiredMoney) {
+				TavernBankrupt();
 				Debug.Log("Required money goal was not reached.");
-				// TODO: Maybe show the score card first?
-				GameOver();
-				//return;
 			}
 
 			scoreCard.UpdateScoreCard(currentRound);
 			scoreCard.Show();
-
-			VikingController.Instance.CanSpawn = false;
-
-			foreach (PlayerComponent player in PlayerManager.Instance.Players) {
-				PlayerInput playerInput = player.GetComponent<PlayerInput>();
-				playerInput.SwitchCurrentActionMap("UI");
-			}
 		}
 
-		private void HandleOnNextRound() {
+		private void EnableGamePlay() {
 			VikingController.Instance.CanSpawn = true;
```

Comment:
Your not checking for the instance here, might want to?

Reply:
Done